app/app.ts

```diff
@@ -21,11 +23,17 @@ export class App {
         return this.server;
     };
 
+    private initRoutes = (): void => {
+        this.server.addRouter(HealthBasePath, healthRouter);
+        this.server.addRouter(ProjectBasePath, projectRouter);
+    };
+
     private initServer = () => {
         this.server.startApp();
     };
 
     private initDB = () => {
+        this.initRoutes();
```

Comment:
You shouldn't be calling initRoutes from initDB, you can call them individually.